tls_ping_client.ml

```diff
@@ -528,7 +528,15 @@ let socks_request_policy server_fingerprint
   | `Socks5 Connect { address = Domain_address address ; port } ->
     Lwt_result.return ({address ; port;
                         username = server_fingerprint }:Socks.socks4_request)
-  | _ -> Lwt_result.fail (`Msg "bad socks conn type")
+  (* FIXME, if connect is to ip address, hostname is hardcoded *)
+  | `Socks5 Connect { address = IPv4_address address ; port } ->
+    Lwt_result.return ({address = "irc.indymedia.org" ; port;
```

Comment:
Hmm, not sure about introducing this part in the code, it seems like something that will be annoying to debug later if we forget about it?

Reply:
it now turns v4 address into string, and fails on v6